client/src/components/EventView/index.js

```diff
@@ -1,7 +1,174 @@
+import ClientService from "../../services";
+import CenteredLoadingSpinner from "../Common/CenteredLoadingSpinner";
+import Stack from "../Common/Stack";
+import Error from "../Common/Error";
+//import PageAccess from "../Common/PageAccess";
 
-const EventView = () => {
-  return <div>Event View</div>;
+// const testEvent = {
+//     eventID: "testID",
+
+// }
+
+const CenterHeading = styled.p`
+    font-size: calc(10px + 2vmin);
+    color: black;
+    font-weight: bold;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+`;
+
+const CenterHeadingLight = styled.p`
+    font-size: calc(7px + 2vmin);
+    color: black;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+`;
+
+const Heading = styled.p`
+    font-size: calc(10px + 2vmin);
+    color: black;
+    font-weight: bold;
+`;
+
+const DisplayStle = styled.div`
+    display: flex;
+    flex-direction: column;
+    justify-content: left;
+    align-items: left;
+    width: 70%;
+    height: 100%;
+    padding: 10% 15%;
+`;
+
+const InviteeRow = styled.div`
+  display: flex;
+  flex-direction: row;
+  //align-items: center;
+  align-items:flex-start;
+  //justify-content: space-around;
+`;
+
+const Main = () => {
+  const [apiStatus, setApiStatus] = useState({
+    loading: false,
+    error: false,
+    message: false
+  });
+
+    const [event, setEvent] = useState({
+        eventName: "testName"
+
+    });
+    useEffect(() => {
+      loadPageData();
+      // eslint-disable-next-line
+    }, []);
+
+    //not sure how to implement this
+    // useEffect(() => {
+    //     const params = new URLSearchParams(event.location.search);
+    //     let eventId = params.get("id");
+    //     if (eventId) {
+    //     loadPageData(eventId);
+    //     } else {
+    //     //redirect to home page since there isn't an eventId
+    //     }
+    //     // eslint-disable-next-line
+    //     }, [location.search]);
+    //please help Morgan!!
+
+  const loadPageData = async () => {
+    setApiStatus({ ...apiStatus, loading: true, error: false });
+    const newApiStatus = { ...apiStatus };
+    try {
+        let eventId = "12346"
+      //const results = await ClientService.events();
+      const results = await ClientService.event(eventId);
+      //TODO: call endpoint for invites
+      if (results.success) {
+        //prepareData(results.events);
+        
+        setEvent(results.event);
+      } else {
+        // default message just in case
+        
+        newApiStatus.error = true;
+        newApiStatus.message = results.message;
+      }
+    } catch (error) {
+      newApiStatus.error = true;
+      newApiStatus.message = error.message;
+    }
+    newApiStatus.loading = false;
+
+    setApiStatus(newApiStatus);
+  };
+
+//const Main = () => {
+  return (
+    <div>
+      <AppBar color={eventPink} name = {event.eventName} date = {event.date}>
```

Comment:
Before this line, you need the <PageAccess /> so that people who don't have an authToken can't access the site. Look at the EventEdit.js if you need help